### DISCUSSION TITLE
喜报

### DISCUSSION THREAD

**Discussion by boomzero** [XMOJ-Script-dev/XMOJ-Script#281] [General]
<img width="663" alt="image" src="https://github.com/XMOJ-Script-dev/XMOJ-Script/assets/85378277/8f9a56a2-0de4-4b53-890f-ba5bcb57a64d">
我不知道为什么我要申请这个，但是……

**Comment by langningchen** [comment]
你又不publish actions

**Comment by boomzero** [comment]
GitHub app
